main.nf

```diff
@@ -28,8 +28,22 @@ def helpMessage() {
 	                                Note: All samples have to be sequenced in one run, otherwise also specifiy "--multipleSequencingRuns"
 	  --FW_primer [str]             Forward primer sequence
 	  --RV_primer [str]             Reverse primer sequence
-	  --metadata [path/to/file]     Path to metadata sheet, when missing most downstream analysis are skipped (barplots, PCoA plots, ...)
-	  --qiime_timezone [str]		Needs to be specified to resolve a timezone error (default: 'Europe/Berlin')
+          --metadata [path/to/file]     Path to metadata sheet, when missing most downstream analysis are skipped (barplots, PCoA plots, ...). 
```

Comment:
I find that description too detailed here, wouldn't that be better located in usage.md?